app/models/registrant.rb

```diff
@@ -2062,6 +2064,13 @@ def extended_status
     end
   end
 
+  def requested_pdf_assistance?
+    if can_request_pdf_assistance? && pdf_delivery && !pdf_is_esigned?
```

Comment:
Lets remove check for can_request_pdf_assistance? because this may be different values when the report is run vs when a registration happens (it depends on state/partner configs in admin UI).